doc/sphinx/source/esmvalcore/preprocessor.inc

```diff
@@ -1,36 +1,218 @@
-:: _preprocessor:
+.. _preprocessor:
 
 ************
 Preprocessor
 ************
-The ESMValTool preprocessor can be used to perform all types of climate data pre-processing needed before indices or diagnostics can be calculated. It is a base component for many other diagnostics and metrics shown on this portal. It can be applied to tailor the climate model data to the need of the user for its own calculations.
+The ESMValTool preprocessor can be used to perform a broad range of operations
+on the input data before diagnostics or metrics are applied. The
+preprocessor performs these operations in a centralized, documented and
+efficient way, thus reducing the data processing load on the diagnostics side.
+
+Each of the preprocessor operations is written in a dedicated python module and
+all of them receive and return an Iris cube, working sequentially on the data
+with no interactions between them.  The order
+in which the preprocessor operations is applied is set by default in order to
+minimize the loss of information due to, for example, temporal and spatial
+subsetting or multi-model averaging. Nevertheless, the user is free to change
+such order to address specific scientific requirements, but keeping in mind
+that some operations must be necessarily performed in a specific order. This is
+the case, for instance, for multi-model statistics, which required the model to
+be on a common grid and therefore has to be called after the regridding module.
+
+In this section, each of the preprocessor modules is described in detail
+following the default order in which they are applied:
+
+* Variable derivation
```

Comment:
Make these links to the relevant sections

Reply:
Done.